docs/source/models/intervention_models/mncnh_pregnancy/oral_iron_antenatal/oral_iron_antenatal.rst

```diff
@@ -387,7 +387,7 @@ Additionally, our methods differ from GBD's in that we estimated two separate GA
 
 2. Find and apply a second shift that replicates the preterm birth dichotomous effect when applied only to the portion of the distribution with baseline gestational age exposures that are *greater* than 32 (the very preterm birth threshold) *minus* this second shift. Note that the second shift will be negative in direction.
 
-`The estimation of the antenatal supplementation gestational age shifts as described was performed in this notebook <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/data_prep/antenatal_interventions/Gestational%20age%20shifts.ipynb>`_
+`The estimation of the antenatal supplementation gestational age shifts as described was performed in this notebook <https://github.com/ihmeuw/vivarium_gates_mncnh/src/vivarium_gates_mncnh/data/ifa_mms_gestation_shifts/ifa_gestational_age_shifts.ipynb>`_
```

Comment:
This link isn't working for me, try https://github.com/ihmeuw/vivarium_gates_mncnh/blob/4c1bf4cef1f2c9d630d41e77ef891b786c5e24d3/src/vivarium_gates_mncnh/data/ifa_mms_gestation_shifts/ifa_gestational_age_shifts.ipynb